### PR TITLE
Disable `schema.db` from Hound review

### PR DESCRIPTION
**Pull request by paoolo**
Due to #86

### CODE REVIEW

**Comment by mkasztelnik** [comment]
Hound defaults are needed to run check locally, right?

**Comment by paoolo** [comment]
@mkasztelnik I don't think so. I did this what was describe in https://github.com/e2/hound-tools/blob/master/README.md

I ran `bundle exec hound-tools init` and commit changes.

**Comment by paoolo** [comment]
I will test how changed Hound settings affect Hound code review in repositories https://github.com/dice-cyfronet/migratio and https://github.com/dice-cyfronet/ip-wrangler

File: .hound.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# Prefer single quotes
-StringLiterals:
-  EnforcedStyle: single_quotes
-  Enabled: true
```
Comment (mkasztelnik) [inline .hound.yml]: Why did you remove these lines?

Comment (mkasztelnik) [inline .hound.yml]: Most probably is should be added into `.hound/overrides.yml` and regenerated.

Comment (paoolo) [inline .hound.yml]: Added in 78a9be2

Comment (mkasztelnik) [inline .hound.yml]: Hmm I see. So now it is added into rubocop defaults but not into hound customization. Did you removed lines above manually or it was done when `bundle exec hound-tools` was executed?

Comment (paoolo) [inline .hound.yml]: I moved it manually.

Comment (mkasztelnik) [inline .hound.yml]: As far as I understand `hound-tools` documentation it does not touch ``hound.yml`, so these changes should be there. What is more information about excluding `schema.rb` should also be added here. 

I don't understand why `hound-tools` cannot take overrides from `.hound.yml` but take them from `.hound/overrides.yml`.

**Comment by mkasztelnik** [comment]
@paoolo I was looking into other projects which use `hound-tools`. It seems like it is not so popular... What is more in documentation I see different files that committed here.

**Comment by paoolo** [comment]
OK. I will check if manual updates in rubocop and hound config will give 
the same result.

**Comment by Nuanda** [comment]
If you intend to add more commits to a branch, remove the ready-for-review label from the PR.

File: .hound.yml (hunk 1 above)
Comment (paoolo) [inline .hound.yml]: I disabled `hound-tools`. I moved this lines to `.rubocop.yml`

Comment (mkasztelnik) [inline .hound.yml]: Ok, but be sure to check if `.rubocop.yml` is loaded by hound.

Comment (paoolo) [inline .hound.yml]: Can I use this changes in [`feature_migrating_virtual_machine_templates`](https://github.com/dice-cyfronet/atmosphere/tree/feature_migrating_virtual_machine_templates)?

Comment (paoolo) [inline .hound.yml]: Testing in #103

**Comment by paoolo** [comment]
@dice-cyfronet/atmo-dev-team updated configuration. Does it fit you?

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+AllCops:
+  RunRailsCops: true
+  Exclude:
+    - db/schema.rb
```
Comment (mkasztelnik) [inline .rubocop.yml]: You can remove this line since @bwilk find that `db/schema.rb` is relevant and it will be remove when #99 is merged.

Comment (paoolo) [inline .rubocop.yml]: Why `db/schema.rb` will be removed from repo?

**Comment by mkasztelnik** [comment]
One small remark reported above, when it is done :+1: from my side :smiley:

File: .rubocop.yml (hunk 2 above)
Comment (mkasztelnik) [inline .rubocop.yml]: It is simply not needed. While updating db in `dummy` project from the spec `spec/dummy/db/schema.rb` is updated no `db/schema.rb`

Comment (paoolo) [inline .rubocop.yml]: Done. Removed.

**Comment by bwilk** [comment]
Second thumb up :+1:
